### PR TITLE
minor changes

### PR DESCRIPTION
**Pull request by anasanasanas**

### CODE REVIEW

**Comment by anasanasanas** [comment]
I signed it!

**Comment by anasanasanas** [comment]
I updated the GitHub account in CLA.

File: jobdispatcher/src/main/java/com/firebase/jobdispatcher/DefaultJobValidator.java [hunk 1]
```diff
@@ -187,13 +187,14 @@ private static int measureBundleSize(Bundle extras) {
 
     @Nullable
     private List<String> validateForPersistence(Bundle extras) {
+        if (extras == null){
```
Comment (ciarand) [inline jobdispatcher/src/main/java/com/firebase/jobdispatcher/DefaultJobValidator.java]: nit: extra space after the )

**Review by ciarand — CHANGES_REQUESTED** [review]
Can you add a test for this?

**Comment by samtstern** [comment]
In April 2019 we announced that Firebase Job Dispatcher would be deprecated today, April 7th 2020.  For this reason we are going to close all open issues and archive the repository.  We will also stop supporting FJD for new app installations (such as those targeting sdk versions greater than Android Q).  Existing apps that rely on FJD should continue to function as usual.

While we will no longer work on FJD, we encourage you to [migrate to Work Manager](https://developer.android.com/topic/libraries/architecture/workmanager/migrating-fb) which is a superior tool and fully supported by the Android team.

Thank you for taking the time to try the library and improve it with this Pull Request.  We sincerely appreciate your effort and we apologize for not addressing it in time.

**Pull request closed without merging**